app/presenters/socure_error_presenter.rb

```diff
@@ -0,0 +1,160 @@
+    I18n.t('components.troubleshooting_options.ipp_heading')
+  end
+
+  def options
+    [
+      {
+        url: help_center_redirect_path(
+          category: 'verify-your-identity',
+          article: 'how-to-add-images-of-your-state-issued-id',
+        ),
+        text: I18n.t('idv.troubleshooting.options.doc_capture_tips'),
+        isExternal: true,
+      },
+      {
+        url: help_center_redirect_path(
+          category: 'verify-your-identity',
+          article: 'accepted-identification-documents',
+        ),
+        text: I18n.t('idv.troubleshooting.options.supported_documents'),
+        isExternal: true,
+      },
+      {
+        url: return_to_sp_failure_to_proof_url(step: 'document_capture'),
+        text: t(
+          'idv.failure.verify.fail_link_html',
+          sp_name: sp_name,
+        ),
+        isExternal: true,
+      },
+    ]
+  end
+
+  def step_indicator_steps
+    Idv::StepIndicatorConcern::STEP_INDICATOR_STEPS
+  end
+
+  private
+
+  attr_reader :error_code, :remaining_attempts, :sp_name, :hybrid_mobile
+
+  SOCURE_ERROR_MAP = {
+    'I848' => 'unreadable_id',
+    'I854' => 'unreadable_id',
+    'R810' => 'unreadable_id',
+    'R820' => 'unreadable_id',
+    'R822' => 'unreadable_id',
+    'R823' => 'unreadable_id',
+    'R824' => 'unreadable_id',
+    'R825' => 'unreadable_id',
+    'R826' => 'unreadable_id',
+    'R831' => 'unreadable_id',
+    'R833' => 'unreadable_id',
+    'R838' => 'unreadable_id',
+    'R859' => 'unreadable_id',
+    'R861' => 'unreadable_id',
+    'R863' => 'unreadable_id',
+
+    'I849' => 'unaccepted_id_type',
+    'R853' => 'unaccepted_id_type',
+    'R862' => 'unaccepted_id_type',
+
+    'R827' => 'expired_id',
+
+    'I808' => 'low_resolution',
+
+    'R845' => 'underage',
+
+    'I856' => 'id_not_found',
+    'R819' => 'id_not_found',
+  }.freeze
+
+  def remapped_error(error_code)
+    SOCURE_ERROR_MAP[error_code] || 'unreadable_id'
+  end
+
+  def heading_string_for(error_code)
+    # i18n-tasks-use t('doc_auth.headers.unreadable_id')
+    # i18n-tasks-use t('doc_auth.headers.unaccepted_id_type')
+    # i18n-tasks-use t('doc_auth.headers.expired_id')
+    # i18n-tasks-use t('doc_auth.headers.low_resolution')
+    # i18n-tasks-use t('doc_auth.headers.underage')
+    # i18n-tasks-use t('doc_auth.headers.id_not_found')
+    I18n.t("doc_auth.headers.#{remapped_error(error_code)}")
+  end
+
+  def error_string_for(error_code)
+    if remapped_error(error_code) == 'underage' # special handling because it says 'Login.gov'
+      I18n.t('doc_auth.errors.underage', app_name: APP_NAME)
+    else
+      # i18n-tasks-use t('doc_auth.errors.unreadable_id')
+      # i18n-tasks-use t('doc_auth.errors.unaccepted_id_type')
+      # i18n-tasks-use t('doc_auth.errors.expired_id')
+      # i18n-tasks-use t('doc_auth.errors.low_resolution')
+      # i18n-tasks-use t('doc_auth.errors.id_not_found')
+      I18n.t("doc_auth.errors.#{remapped_error(error_code)}")
+    end
```

Comment:
```suggestion
    remapped_error_code = remapped_error(error_code)
    if rremapped_error_code == 'underage' # special handling because it says 'Login.gov'
      I18n.t('doc_auth.errors.underage', app_name: APP_NAME)
    else
      # i18n-tasks-use t('doc_auth.errors.unreadable_id')
      # i18n-tasks-use t('doc_auth.errors.unaccepted_id_type')
      # i18n-tasks-use t('doc_auth.errors.expired_id')
      # i18n-tasks-use t('doc_auth.errors.low_resolution')
      # i18n-tasks-use t('doc_auth.errors.id_not_found')
      I18n.t("doc_auth.errors.#{remapped_error_code}")
    end
```